tgui/packages/tgui/interfaces/common/JobToIcon.ts

```diff
@@ -0,0 +1,6 @@
+// Used to handle jobs that don't have a trim.
+export const JobToIcon = {
```

Comment:
Remove this file entirely, and please just port orbit icons, even if they're used for just this menu.

I had a quick glance at the menu without the icons, and it doesn't look nice to me.

See yogstation's port of this to see what you'd need to do. It's surprisingly little.

Reply:
I hate this file, but this can be removed for DM-sided everything later.